src/data-card-advanced.tsx

```diff
@@ -0,0 +1,460 @@
+                )}
+
+                {description && (
+                    <Text2
+                        truncate={descriptionLinesMax}
+                        as="p"
+                        regular
+                        color={vars.colors.textSecondary}
+                        hyphens="auto"
+                        aria-label="Texto"
+                    >
+                        {description}
+                    </Text2>
+                )}
+            </Stack>
+        </div>
+    );
+};
+
+type CardFooterProps = {
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+};
+
+const CardFooter: React.FC<CardFooterProps> = ({
+    button,
+    footerImage,
+    footerText,
+    footerTextLinesMax,
+    buttonLink,
+}) => {
+    const hasButton = !!button;
+    const hasFooterImage = !!footerImage;
+    const hasFooterText = !!footerText;
+    const hasButtonLink = !!buttonLink;
+    const hasAllItens = hasButton && (hasFooterImage || hasFooterText) && hasButtonLink;
+
+    const flexDirection = hasAllItens ? 'column' : 'row';
+    const alignItems = hasAllItens ? 'start' : 'center';
+    const marginTop = hasAllItens ? '8px' : '16px';
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <div>
+            <NegativeBox>
+                <Divider />
+            </NegativeBox>
+
+            <div className={styles.actions} style={{flexDirection, alignItems}}>
+                {hasButton && (
+                    <div
+                        aria-hidden="true"
+                        onClick={(event) => event.stopPropagation()}
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        style={{marginTop: '16px', marginRight: '16px', position: 'relative', zIndex: '2'}}
+                    >
+                        {button}
+                    </div>
+                )}
+                <div style={{display: 'flex', flexDirection: 'row', marginTop, flexWrap: 'wrap'}}>
+                    {hasFooterImage && (
+                        <div
+                            style={{marginRight: '16px', zIndex: '0'}}
+                            className={sprinkles({alignItems: 'center', display: 'flex'})}
+                        >
+                            {footerImage}
+                        </div>
+                    )}
+
+                    {hasFooterText && (
+                        <div style={{maxWidth}} className={styles.footerText}>
+                            <Text2 truncate={footerTextLinesMax} regular>
+                                {footerText}
+                            </Text2>
+                        </div>
+                    )}
+                </div>
+                {hasButtonLink && (
```

Comment:
buttonLink is not aligning when falling off line

![image](https://github.com/Telefonica/mistica-web/assets/6722153/a740b79b-8085-45af-9556-612d9a2fa63d)

Reply:
visually misaligned, yes. The transparent background cause this problem. ButtonLink has a property called `aligned` to solve this problem. When the buttonLink falls, buttonLink has to use this prop.

https://tinyurl.com/29pldjsn

Reply:
unfortunately I can't put this property directly, because the component only receives the buttonLink, what I could do is copy the behavior of this prop, but that would leave the component misaligned when hovering, unless it has its own animation to fix it back in place smoothly

Reply:
Is it really necessary to correct this alignment? after I realized that the current DataCard also has this problem:

![MicrosoftTeams-image (4)](https://github.com/Telefonica/mistica-web/assets/123959276/bc4fa533-f80b-40a4-b6ee-b2fde4c2cd34)

Reply:
Sorry for so many messages, I understood that the component will be used more in the mobile context so it won't have a hover, so your proposal to use the Aligned prop is quite valid, but since the data-card only receives the buttonLink, it will be the programmer who will have to put this prop, when passing the buttonLink.

Reply:
In this screenshot, the link is aligned
![image](https://github.com/Telefonica/mistica-web/assets/6722153/f8d11141-7c37-40b3-9336-9494ffb58d95)

also buttonGroup has the logic to include this behavior
[example](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8ARAhgFzQYTQJygB0A7AAlIEtJiBeYE88pbCvMAGxlICM0wBrAOZ4IAV2JRsEdhDx1IMvAGcAdNzxoJAGQgB3AL6klFAF4w6AFgAM%2BgHwNGTAJLUAygAsIABwBKMQaLs%2BKQKsvLSsqrqmlCGAPT2ZIxIcSxsnInk%2Bg7uMGhQ7BTE5sBIACpogqQYAJ5e5oQgXiIAthCNtgASeQVFMCkVgrbZSRgUGJw0jWXjnI0OSqLcYxMNIK5LK3MgDrBKYHgUXmMQtI0oMPuHxxSn80kwAB4YGnRIAApBYDCe7LB4pASI3I3FEGAwpzoDiYACEwRDiO9Di18DUjCj2OxSKckZclHQABQASlINFspGA%2Bjs0MYAEEwCcyEooGglKRWVB2cyuZyOTyeRy0DSUnDwTjkajMqRgTx4actEV%2BG9RQiFcR%2BNjEXg8YSSWSKVTbGr%2BCK5cRjcMSAkSCAADQgDC5FqXBAAbRAAFkIAA3ChKLB4EAAXXtugoUEdSjdAGYAGxWIP6IA)

Reply:
Related to this, when the link is on the right, seems to be not aligned as same as specs 
![image](https://github.com/Telefonica/mistica-web/assets/6722153/a05b2284-b4b1-46e1-a49b-ffefd2adb03e)

Maybe is missing `aligned` prop in this case too